setup/system.sh

```diff
@@ -144,6 +144,9 @@ pollinate  -q -r
 
 # Between these two, we really ought to be all set.
 
+echo 'Creating SSH public key... (might take a while)'
```

Comment:
A few things:
* We should skip this if the file exists, using `if [ ! -f /root/.ssh/id_rsa ]; then . . . fi`.
* 2048 bits is plenty, which is the default, so `-b 8192` can be dropped.
* I wonder if we should generate this in a different location and specify it explicitly when executing rsync, since it's not really an identification of the root user per se. It's more of an identification of the machine's Mail-in-a-Box management scripts.